### DISCUSSION TITLE
گرفتن پسوورد ایمیل جایگزین درگوشی سرقتی

### DISCUSSION THREAD

**Discussion by Mahbobehrohollahzadeh** [community/community#138107] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

ایمیل گوشی سرقت شده من از جی میل به example.com عوض شده ومن میخواهم پسوورد ایمیل جدید را پیداکنم

**Comment by ebndev** [comment]
Hi @Mahbobehrohollahzadeh, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
